proxy-node-acceptance-tests/features/step_definitions/user_steps.rb

```diff
@@ -87,6 +87,10 @@ def stub_idp_user
   assert_text('GB/EU/_tr_')
 end
 
+And('they should have a response issued by the Proxy Node') do
+  assert_text(env('PROXY_NODE_URL') + '/ServiceMetadata')
```

Comment:
Maybe: `assert_text('Issuer: ' + env('PROXY_NODE_URL') + '/ServiceMetadata')` to be a bit sure it's appearing as expected?

Reply:
Good call, I'll change this